gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -241,6 +243,11 @@ public Boolean load(String bucketName) throws Exception {
   // Function that generates downscoped access token.
   private final Function<List<AccessBoundary>, String> downscopedAccessTokenFn;
 
+  // Google root logger, logger that spans over the <package_shading_prefix>.com.google.* coverage.
+  @VisibleForTesting
+  protected static final Logger googleLogger =
+      Logger.getLogger(GoogleLogger.class.getName().replaceAll("(com\\.google).*", "$1"));
+
```

Comment:
I wonder if we actually need to do this, I think it will be cleaner to implement utility class that has a method similar to this: https://github.com/google/flogger/blob/flogger-0.5.1/api/src/main/java/com/google/common/flogger/LoggerConfig.java#L129-L138